pyproject.toml

```diff
@@ -95,7 +91,7 @@ cov = "pytest --cov-report=xml --cov-config=pyproject.toml --cov=src/dysh --cov=
 
 # Run tests across all supported version of Python
 [[tool.hatch.envs.test.matrix]]
-python = ["39", "310", "311", "312"]
+python = ["310", "311", "312", "313"]
```

Comment:
Should 313 be removed here?

Reply:
I think something got reverted during the rebase onto main. Will look tomorrow